pyproject.toml

```diff
@@ -213,3 +213,14 @@ cli = 'vendored-meson/meson/meson.py'
   ".spin/cmds.py:notes",
 ]
 "Metrics" = [".spin/cmds.py:bench"]
+
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git*,*.pdf,*.svg,*.css,.mailmap,./doc/changelog'
+check-hidden = true
+# some cases + pragma to add to line to ignore entire line
+# Assume that capitalized shortish (up to 10) Word is a name - skip
+# Skip all shortish (up to 5) ACRONYMS
+ignore-regex = '"worl"|https://citeseerx\.ist\.psu\.edu|\b([A-Z][a-z]{1,9}|[A-Z]{1,5})\b|.*codespell-ignore.*|/seh/'
+# some common hits and some "questionable" and false positives (rare words)
+ignore-words-list = 'nd,nin,numers,numer,inout,ccompiler,fo,CCompiler,dout,ans,ser,ot,asign,siz,alow,fro,delimitor,alo,dum,desig,recuse,implementor'
```

Comment:
Can this be folded into a multiline statement?

Reply:
Ha -- it seems to can. Splitting into multiple lines